usage/camera/installing.md

```diff
@@ -2,9 +2,9 @@
 
 ## Connecting the cameras
```

Comment:
Should this be "## Connecting the camera" (i.e. singular) ?

Reply:
Done.